modules/sqs/main.tf

```diff
@@ -33,6 +43,98 @@ data "aws_iam_policy_document" "status_update_queue_policy" {
   }
 }
 
+## internal-report-queue - get_current_archive_list lambda reads from this queue to get what it needs to pull in next
+## ==================================================================================================================
+resource "aws_sqs_queue" "internal_report_queue" {
+  ## OPTIONAL
+  name                       = "${var.prefix}-internal-report-queue"
+  delay_seconds              = var.sqs_delay_time_seconds
+  max_message_size           = var.sqs_maximum_message_size
+  message_retention_seconds  = var.internal_report_queue_message_retention_time_seconds
+  tags                       = local.tags
+  policy                     = data.aws_iam_policy_document.internal_report_queue_policy.json
+  visibility_timeout_seconds = 10800 # Set to 12 times lambda max time  # todo: Pass in the variable.
```

Comment:
Yeah, wasn't confident in this value. Our standard for workflow retries is `"MaxAttempts": 6` and there are two lambdas, so theoretically it could take 12 timeouts for the workflow to definitively fail. I'd be in favor of dropping the workflow retries if that's what sounds good.

Reply:
Will lower to 3 retries

Reply:
Lower timeout beyond processing time